### DISCUSSION TITLE
second harvester

### DISCUSSION THREAD

**Discussion by MagicHuman** [Chia-Network/chia-blockchain#5955] [Support]
There was a main harvester at work, I sowed two hard 10 TB each and looked for evidence, then I assembled another harvester and put it at home. on the fact that everything is fine at work, I am looking for and sowing 16 TB, and on a harvester (home) I transferred the seeded 2x10 TB, but the point is, as I understood the home harvester does not mine, but only fills, I tried to combine, but it did not work. The question is what, how to make the harvester 2 (at home) become the main one, farm and grow, and the worker only sows? thanks in advance.

**Comment by lvpcguru** [comment]
[the wiki](https://github.com/Chia-Network/chia-blockchain/wiki/Farming-on-many-machines)

**Comment by BrandtH22** [comment]
@MagicHuman , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
